tuist/tuist
# Support for defining a dependency file in run script phases

Resolves https://github.com/tuist/tuist/issues/4939

### Short description 📝

Expose `dependencyFile` property that exists in `PBXShellScriptBuildPhase` by adding it to `Tuist.TargetScript` and `ProjectDescription.TargetScript`.

### How to test the changes locally 🧐

Using the new property in any of the `.pre` - `.post` functions, it should set the value in Xcode build phase.

### Contributor checklist ✅


### Reviewer checklist ✅
